phpunit.xml.dist

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
>  If someone wants to use their own settings they can modify the phpunit.xml in their own local checkout.

That's the whole point of having phpunit.xml.dist and having phpunit.xml on the .gitignore list.

Can you elaborate more?